doc/Language/classtut.pod6

```diff
@@ -192,69 +100,72 @@ X<|classes,has>
 X<|behavior>
 X<|classes,behavior>
 
-Raku, like many other languages, uses the C<class> keyword to define a
-class. The block that follows may contain arbitrary code, just as with
-any other block, but classes commonly contain state and behavior
-declarations. The example code includes attributes (state), introduced
-through the C<has> keyword, and behaviors, introduced through the C<method>
-keyword.
-
-X<|type object>
-X<|DEFINITE>
-X<|.DEFINITE>
+The following piece of code implements a dependency handler. It's not a lot of code, and yet the result is interesting and useful.
+It will be used as an example throughout the following sections.
 
-Declaring a class creates a new I<type object> which, by default, is installed
```

Comment:
Removing index anchors is definitely a no-no. They probably have moved elsewhere but, again, we need to approach stuff piecewise so that we can judge every issue by itself.